pkg/pipelines/scm/utility.go

```diff
@@ -0,0 +1,109 @@
+package scm
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"strings"
+
+	pipelinev1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+
+	triggersv1 "github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+)
+
+func GetDriverName(rawURL string) (string, error) {
+
+	u, err := url.Parse(rawURL)
+	if err != nil {
+		return "", err
+	}
+	if s := strings.TrimSuffix(u.Host, ".com"); s != u.Host {
+		return strings.ToLower(s), nil
+	}
+
+	if s := strings.TrimSuffix(u.Host, ".org"); s != u.Host {
```

Comment:
I'm not sure that either GitHub or GitLab offer `.org` hosted repos?

But it feels like this, and `GetRepoName` are both undertested?

Reply:
Yeah. `.org` is not supported. I've removed the check for that and added unit tests for `getRepoName` and `getDriverName`